### ISSUE TITLE
Vega 64 - Cannot build DAG on Linux

### ISSUE THREAD

**Issue by joshuaguild** [nanopool/Claymore-Dual-Miner#87]
Linux v. 16.04.3
Driver - latest amdgpu-pro 17.30
Tried with and without ROCM optional support.

Risers, PSU, and PCIe slots are good. When running Claymore, sometimes the card is recognized and sometimes it is not. Ends up failing and cannot build DAG unfortunately. 

Happy to provide more details to help in research.

**Comment by joshuaguild** [comment]
Could be that ROCM requires PCI 3.0 and I'm running from 2.0 slots, will continue to explore.

**Comment by joshuaguild** [comment]
Nope, not the PCI slot. The driver still works with my 470s but not with my Vega so I'm not sure what it is at this point. Either the device isn't recognized or it shows as a Vega but is classified as a different card (GTX980 IIRC).

Let me know if I can provide any more information to help with fixing this.

**Comment by zyxelenator** [comment]
Same here, as of Now VEGA 64 is not working in Linux. Tried other mining software using Simple Mining OS. Vega 64 is either not detected or it is locking up.
